apps/explorer/config-default.yaml

```diff
@@ -0,0 +1,317 @@
+#######################################################################################
+##                                  DEFAULT CONFIG                                   ##
+#######################################################################################
+# Defines the default configuration
+#   🚫 WARNING 🚫: Do not modify this file
+#       - For custom config create a new file "custom/config.yaml"
+#       - You can override in the custom file any config that is defined here
+#       - For redefining components, or for more information, follow the instructions in ./README.md
+#   ⚙️Read more here:
+#       https://github.com/gnosis/gp-v1-ui/wiki/Config
+
+# App Id
+#   Identify the app. The IDs range from 1 until 256
+#   Every transaction sent to the blockchain will include the ID and some basic analytic info
+#
+#   ⚠️IMPORTANT: The ID must be unique per App/Client of Gnosis Protocol:
+#       https://github.com/gnosis/gp-v1-ui/wiki/App-Ids-for-Forks
+#   More information about forking the web:
+#       https://github.com/gnosis/gp-v1-ui/wiki/Fork-project
+#
+#appId: 1
+
+# Name of the app
+name: Gnosis Protocol v1 Web
+
+# Logo used for the favicon
+logoPath: './src/assets/img/CowProtocol-icon.svg'
+
+# HTML template
+templatePath: './src/html/index.html'
+
+# Whitelisted tokens from Gnosis Contract retrieved from a smart contract
+tcr:
+  type: 'multi-tcr' # choices: multi-tcr | none
+  config:
+    lists:
+      - networkId: 1
+        listId: 1
+        # contractAddress: '0x1854dae560abb0f399d8badca456663ca5c309d0' # Testing TCR with rare tokens
+        contractAddress: '0x93DB90445B76329e9ed96ECd74e76D8fbf2590d8'
+
+      - networkId: 4
+        contractAddress: '0xBb840456546496E7640DC09ba9fE06E67C157E1b'
+
+# Price estimation/orderbook graph
+dexPriceEstimator:
```

Comment:
i don't think we use this

Reply:
Addressed here: https://github.com/cowprotocol/cowswap/pull/3593